### DISCUSSION TITLE
How to have a single table for an entity with a value obj attribute using imperative mapping?

### DISCUSSION THREAD

**Discussion by jalvespinto** [sqlalchemy/sqlalchemy#7788] [Usage Questions]
Lets say I have the following classes:
```
class Child:

    att1: str
    att2: str

class Parent:

    attr1: str
    child: Child
```
Is there a way I can have a parents table with columns like `attr1`, `child_attr1` and `child_attr2` using imperative? I know how to do it using relationships and a dedicated table for child, but is there a way to place them in a single row like that? Tks

**Comment by zzzeek** [comment]
There's a few different things you might be asking here, but if you want there to be one "parent" table, no "child" table, but you want to have Parent->Child where Child is an object, that's available using the [composite](https://docs.sqlalchemy.org/en/14/orm/composites.html) construct.  I think that's what you're asking for.
